conjur/login/login_controller.py

```diff
@@ -25,6 +25,8 @@ class LoginController:
 
     def __init__(self, ssl_verify, user_password, credential_data, login_logic):
         self.ssl_verify = ssl_verify
+        # We check if the user supplied --insecure because
+        # this command does not go through the Client
```

Comment:
Sorry, I didn't get the prupose of this comment nor its meaning...
Could you better explain?

Reply:
Client (client.py) when initialized has this check (if not ssl_verify, print an insecure message). For this command (init and login/logout) we don't initialize the client because we don't want to send an request to Conjur yet. Therefore, we need to have this check out here

Reply:
Client (client.py) when initialized has this check (if not ssl_verify, print an insecure message). For these commands (init and login/logout) we don't initialize the client because we don't have the needed information to be successful yet. Therefore, we need to have this check out here